lib/Events/AbstractFormEvent.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace OCA\Forms\Events;
+
+use OCA\Forms\Db\Form;
+use OCP\EventDispatcher\Event;
+use OCP\EventDispatcher\IWebhookCompatibleEvent;
+
+abstract class AbstractFormEvent extends Event implements IWebhookCompatibleEvent {
+	protected Form $form;
+
+	public function __construct() {
+		parent::__construct();
+	}
```

Comment:
```suggestion
	public function __construct(
		protected Form $form,
	) {
		parent::__construct();
	}
```
What is the reason for not setting this property in the constructor here rather than in the child class?